pkg/config/loader_test.go

```diff
@@ -0,0 +1,305 @@
+package config
+
+import (
+	"os"
+	"reflect"
+	"strconv"
+	"testing"
+	"time"
+
+	autoheal "github.com/openshift/autoheal/pkg/apis/autoheal/v1alpha2"
+	batch "k8s.io/api/batch/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
I think that in other places we use `core` instead of `v1`, can you do the same?

Reply:
This is actually `v1` in `loader.go` so I kept it the same for consistency of testing..